daemon/daemon.go

```diff
@@ -905,6 +883,51 @@ func (d *Daemon) init() error {
 	return nil
 }
 
+// startLXCMapSync spawns a controller which synchronizes the host entries in BPF lxc Map
+// with the in-memory host IP address values.
+func (d *Daemon) startLXCMapSync() error {
+	// Initially, clean all endpoint entries
+	if err := lxcmap.LXCMap.DeleteAll(); err != nil {
+		return err
+	}
+
+	controller.NewManager().UpdateController("lxcmap-bpf-host-sync",
+		controller.ControllerParams{
+			DoFunc: func() error {
+				// TODO: Update addresses first, in case node addressing has changed.
+				// TODO: Once these start changing on runtime, figure out the locking strategy.
+				// TODO: Delete stale host entries from the lxcmap.
```

Comment:
If you're planning to bind lxcmap update to ipcache update, then ipcache update already has GC, so maybe you could put it there..? (See `OnIPIdentityCacheGC()`).

Reply:
`OnIPIdentityCacheGC()` deletes stale entries from ipcache bpf map. Here we are adding potentially missing entries. I agree that these should be combined into a more comprehensive desired/realized state implementation, but that is out of scope for this PR.